archive_db.py

```diff
@@ -22,15 +22,19 @@
 logger.info("** archive-db.py **")
 
 OUT_PATH = ENV.get('OUT_PATH', 'data')
-
 DB_PARAMS = ENV['MYSQL_DATABASE']
-ENGINE = create_engine(
-    f"mysql://{DB_PARAMS['USER']}" + 
+SSL_CA_PATH = ENV.get('SSL_CA_PATH', None)
+
+conn_str = (
+    'mysql' + 
+    f"://{DB_PARAMS['USER']}" + 
     f":{DB_PARAMS['PASSWORD']}" + 
     f"@{DB_PARAMS['HOST']}" + 
     f":{DB_PARAMS['PORT']}" + 
     f"/{DB_PARAMS['DATABASE']}?charset=utf8"
 )
+
+ENGINE = create_engine(conn_str, connect_args={'ssl': {'ca': SSL_CA_PATH}}) if SSL_CA_PATH else create_engine(conn_str)
```

Comment:
Can we separate out this one line content into different lines, for readability purposes?

Reply:
Sure, I actually had that before, I can change it back.

Reply:
Should be better now.